git.mk

```diff
@@ -120,6 +120,8 @@ GITIGNORE_MAINTAINERCLEANFILES_M4_LIBTOOL = \
 			ltsugar.m4 \
 			ltversion.m4 \
 			lt~obsolete.m4 \
+			ax_recursive_eval.m4 \
+			libgcrypt.m4 \
```

Comment:
These should be added here:
https://github.com/tpm2-software/tpm2-tss/blob/9f0584a1e093d9aa0b37b9e3b17359d372b4b43d/Makefile.am#L27-L50